static/app/components/replays/breadcrumbs/utils.tsx

```diff
@@ -73,6 +131,14 @@ export function getTitle(crumb: Crumb) {
     return crumb.message ?? crumb.description;
   }
   const [type, action] = crumb.category.split('.') || [];
+
+  if (action === 'slowClickDetected') {
+    if ((crumb.data as Record<string, unknown> | undefined)?.endReason === 'timeout') {
```

Comment:
🤮 but I want to get this out so we can more easily debug dead clicks in replays.